### DISCUSSION TITLE
Standard edition not installed

### DISCUSSION THREAD

**Discussion by clarknelson** [solspace/craft-freeform#1148] [Questions & Answers] [answered]
**Describe the bug or issue you're experiencing**

This message has appeared prompting me to install the standard version.

![CleanShot 2024-03-13 at 14 24 48@2x](https://github.com/solspace/craft-freeform/assets/4106671/9a463cd0-2cf4-4cc9-807b-c53605f8f21d)

When I go to resolve there doesnt appear to be a standard version. We should also be licensed for the pro version.

![CleanShot 2024-03-13 at 14 25 43@2x](https://github.com/solspace/craft-freeform/assets/4106671/4f839b6d-9be1-4c73-890a-ca33b1d8444a)

Nothing has changed in the project-config regarding the key or the version. 

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.8.2
- Freeform Version: 4.2.7
- Freeform Edition: Pro
- Fresh Install or Upgrade:

**Comment by kjmartens** [comment, marked as answer]
Sorry for the trouble you're experiencing, @clarknelson.

Is this issue still happening for you? I am not sure this is something we have control over, unless of course your project config file shows `standard` for the Freeform edition (which could then be updated to `pro`).

If that's not the case, your best bet is to contact the Craft team for assistance with this. 🙂
